db/migrate/20150612104023_add_gov_response_at_to_petitions.rb

```diff
@@ -0,0 +1,5 @@
+class AddGovResponseAtToPetitions < ActiveRecord::Migration
+  def change
+    add_column :petitions, :gov_response_at, :timestamp
```

Comment:
Yeah, I just copied the name from the story ACs. `parliament_response_at` sounds good to me, I will change it. :+1: